internal/ui/table.go

```
@@ -409,6 +423,24 @@ func getFormatFallbackChain(primary TimeAxisFormat) []TimeAxisFormat {
 			fallbacks = append(fallbacks, TimeAxisFormat{"15", 4 * time.Hour, "hours"})
 		}
 
+		return fallbacks
+	} else if duration >= 24*time.Hour && duration < 7*24*time.Hour {
+		// For daily intervals, provide fallback options
+		fallbacks := []TimeAxisFormat{primary}
+		
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)